packages/core/src/routes/init.ts

```diff
@@ -135,9 +135,8 @@ const createRouters = (tenant: TenantContext) => {
 
 export default function initApis(tenant: TenantContext): Koa {
   const apisApp = new Koa();
-
   const { adminUrlSet, cloudUrlSet } = EnvSet.values;
-  apisApp.use(koaCors(adminUrlSet, cloudUrlSet));
+  apisApp.use(koaCors([adminUrlSet, cloudUrlSet], ['/profile', '/verifications']));
```

Comment:
nit: better use a share const for these account API prefixes

Reply:
good idea, I'll do it in the PR of renaming `/profile` to `/account`